projects/lima/2025-lima-incubation.md

```diff
@@ -0,0 +1,414 @@
+
+Intuit considers that if ever needed it would be hard to find a similarly smooth solution and would consider moving to a cloud based solution for developer environments instead of maintaining a fork.
+
+###### Strengths and Improvements
+
+For Intuit the main strength of both Rancher Desktop and Lima is the ability to achieve a similar solution to that of Docker Desktop based purely on open source tools. This also brought significant savings when accounting to the number of users and licensing costs of their previous solution.
+
+Other strengths include the reliability and stability and the fact that no issues have been seen in production. The project community is small but very active on slack.
+
+Intuit believes the project would benefit from additional awareness and better marketing and that it should focus more on enterprise roll-out solutions (particularly for Rancher Desktop).
+
+##### Abiosoft
+
+Usage of Lima in this context is done via Colima. The interview was done with the author of Colima, Abiola Ibrahim.
+
+###### Usage
+
+Colima has been relying on Lima since 2020, with the main motivation coming from one of the core features of the project - the ability to run headless virtual machines on macOS. Podman and Docker were also checked as alternatives.
+
+Colima does not yet have a 1.0 release. It relies on the latest available version of Lima, and acts as an early tester of new releases. Colima plans to follow on the recently added Windows support, and add similar support directly in Colima.
+
+The authors of Colima contributed the support for nested virtualization in Lima.
+
+###### Perception and Feeling
+
+Conversations with the Lima project have been very productive from the early start, with direct communication with the maintainers of the Lima project. Minor challenges during adoption included support for file watchers (originally with little interest from Lima, but later included); support for network providers and integration with Colima, with good ideas coming from replicating available functionality in Podman.
+
+GitHub issues are the main source for communication, complemented by slack. Very happy with the support and level of response from the Lima maintainers. It appears the project is also gaining popularity, and Abiola gives as an example the multiple contributions to the project coming from GSOC internships.
+
+###### Strengths and Improvements
+
+Abiola sees the clear instructions and ease of contributions as a main strength of the project, together with it being an unique project offering virtual machine support for Mac environments.
+
+Suggested improvements to the project include maturing the Windows support, as not all features are currently available across all platforms. Not a project weakness but expected as the project matures. Recent changes to cleanup the distribution channels has also slightly complicated the setup on Mac environments, with tools such as qemu having to be installed independently.
```

Comment:
I don't get this sentence.
We have never shipped Lima with QEMU, and QEMU is no longer needed for most users, as vz (Virtualization.framework) is used by default since Lima v1.0.